### PR TITLE
Update to USWDS 2.12.2

### PR DESCRIPTION
**Pull request by thisisdano**

### CODE REVIEW

**Comment by thisisdano** [comment]
Circle seems broken, unrelated to this release. Build properly in Federalist. [Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/use-2.12.2/) →

**Comment by mejiaj** [comment]
That error is so weird. The error complains about a dependency, but I don't see it locally.

File: pages/documentation/getting-started/fundamentals.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Install USWDS from the command line, and save it as a dependency in your `packag
 npm install uswds --save
 
 # [a lot of notices]
-# + uswds@2.12.1 [or another version number]
+# + uswds@2.12.2 [or another version number]
```
Comment (mejiaj) [inline pages/documentation/getting-started/fundamentals.md:95]:
```suggestion
# + uswds@{{ site.uswds_version }} [or another version number]
```

Will give us the value from `uswds_version` in `_config.yml:5`. This way we don't have to keep manually updating.

File: pages/documentation/getting-started/fundamentals.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ npm will show some notifications, install USWDS, and display the version number
 {:.site-terminal}
 ```json
 "dependencies": {
-  "uswds": "^2.12.1" [or another version number]
+  "uswds": "^2.12.2" [or another version number]
```
Comment (mejiaj) [inline pages/documentation/getting-started/fundamentals.md:104]:
```suggestion
  "uswds": "^{{ site.uswds_version }}" [or another version number]
```

Same here.

**Review by mejiaj — COMMENTED** [review]
Thoughts on using `uswds_version` to set it on pages like _Fundamentals_?

We could also use it on `download-buttons-design.html` for the link text.

File: _includes/download-buttons-design.html [hunk 3]
```diff
@@ -5,8 +5,8 @@
 <div class="link_group-download">
   <a class="link-download"
     href="https://github.com/uswds/uswds-for-designers/releases/download/v2.5.0/uswds-for-designers-v2.5.0.zip"
-    onclick="ga('send', 'event', 'Downloaded design files - Sketch', 'Clicked download design files button inside site');">Download the USWDS 2.12.1 Design Kit (Sketch) [ZIP; 13.2 MB]</a>
+    onclick="ga('send', 'event', 'Downloaded design files - Sketch', 'Clicked download design files button inside site');">Download the USWDS 2.12.2 Design Kit (Sketch) [ZIP; 13.2 MB]</a>
```
Comment (mejiaj) [inline _includes/download-buttons-design.html:8]:
```suggestion
    onclick="ga('send', 'event', 'Downloaded design files - Sketch', 'Clicked download design files button inside site');">Download the USWDS {{ site.uswds_version }} Design Kit (Sketch) [ZIP; 13.2 MB]</a>
```

File: _includes/download-buttons-design.html [hunk 4]
```diff
@@ -5,8 +5,8 @@
...
   <a class="link-download"
     href="https://github.com/uswds/uswds-for-designers/releases/download/v2.5.0/uswds-for-designers-v2.5.0.zip"
-    onclick="ga('send', 'event', 'Downloaded design files - Adobe XD', 'Clicked download design files button inside site');">Download the USWDS 2.12.1 Design Kit (Adobe XD) [ZIP; 13.2 MB]</a>
+    onclick="ga('send', 'event', 'Downloaded design files - Adobe XD', 'Clicked download design files button inside site');">Download the USWDS 2.12.2 Design Kit (Adobe XD) [ZIP; 13.2 MB]</a>
```
Comment (mejiaj) [inline _includes/download-buttons-design.html:11]:
```suggestion
    onclick="ga('send', 'event', 'Downloaded design files - Adobe XD', 'Clicked download design files button inside site');">Download the USWDS {{ site.uswds_version }} Design Kit (Adobe XD) [ZIP; 13.2 MB]</a>
```

**Comment by thisisdano** [comment]
Merging over the unrelated Circle error.

**Pull request merged**